### PR TITLE
User list, detail and tag list autoupdates

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
TODO: there is still an issue...

File: client/src/app/site/tags/components/tag-list/tag-list.component.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { MatDialog, MatDialogRef } from '@angular/material/dialog';
 import { TranslateService } from '@ngx-translate/core';
 import { PblColumnDefinition } from '@pebula/ngrid';
 
-import { SimplifiedModelRequest } from 'app/core/core-services/model-request-builder.service';
+import { SimplifiedModelRequest, DEFAULT_FIELDSET } from 'app/core/core-services/model-request-builder.service';
```
Comment (tsiegleauq) [inline client/src/app/site/tags/components/tag-list/tag-list.component.ts:8]: > DEFAULT_FIELDSET

can this be removed? They are usually in the repositories

Comment (FinnStutzenstein) [reply client/src/app/site/tags/components/tag-list/tag-list.component.ts:8]: yes, right.

Comment (tsiegleauq) [reply client/src/app/site/tags/components/tag-list/tag-list.component.ts:8]: welp matters little. Cleanup will eventually come :D

**Pull request merged**